cmd/geth/config.go

```diff
@@ -63,11 +64,19 @@ var tomlSettings = toml.Config{
 		return field
 	},
 	MissingField: func(rt reflect.Type, field string) error {
-		link := ""
-		if unicode.IsUpper(rune(rt.Name()[0])) && rt.PkgPath() != "main" {
-			link = fmt.Sprintf(", see https://godoc.org/%s#%s for available fields", rt.PkgPath(), rt.Name())
+		id := fmt.Sprintf("%s.%s", rt.String(), field)
+		if deprecated(id) {
+			log.Warn("Config field is deprecated and won't have an effect", "name", id)
+			// Don't fail
+			return nil
+		} else {
+			link := ""
+			fmt.Printf("hereee %v %s\n", rt.String(), field)
```

Comment:
:)

Reply:
Once this is gone, I think we can merge this, and then update #23118 to actually make use of this feature, and get that one in aswell

Reply:
Ops :) sorry will try to be more careful next time